src/targets/awsLambda.ts

```diff
@@ -0,0 +1,139 @@
+import { logger as loggerRaw } from '../logger';
+import { TargetConfig } from '../schemas/project_config';
+import { BaseTarget } from './base';
+import {
+  BaseArtifactProvider,
+} from '../artifact_providers/base';
+import { ConfigurationError, reportError } from '../utils/errors';
+import Lambda = require('aws-sdk/clients/lambda');
+import fs = require('fs');
+
+const logger = loggerRaw.withScope(`[aws-lambda]`);
+
+/**
+ * RegExp for the AWS Lambda package.
+ * The pattern matches the following structure:
+ * `sentry-node-serverless-{version}.zip`.
+ */
+const DEFAULT_AWS_LAMBDA_DIST_REGEX = /sentry-node-serverless-\d+(\.\d+)*\.zip$/
+
+const awsAllRegions = [
+  'us-east-1',
+  'us-east-2',
+  'us-west-1',
+  'us-west-2',
+  'ap-south-1',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'ap-northeast-1',
+  'ap-northeast-2',
+  'ca-central-1',
+  'eu-central-1',
+  'eu-west-1',
+  'eu-west-2',
+  'eu-west-3',
+  'sa-east-1',
+];
+
+const layerName = 'SentryNodeServerlessSdk';
```

Comment:
@iker-barriocanal It would be great if we could make this some sort of environment variable or something so that when we test this locally, we can upload it to a different layer without modifying the code. But we can default to `SentryNodeServerlessSdk` if no env variable is set.